AutomatedTesting/Gem/PythonTests/scripting/VariableManager_ExposeVarsToComponent.py

```diff
@@ -46,9 +46,10 @@ def VariableManager_ExposeVarsToComponent():
 
     general.idle_enable(True)
 
-    # 1) Open Script Canvas window
-    qtpy_o3de_editor = QtPyO3DEEditor()
+    # 1) Create a new level file and then open Script Canvas editor
```

Comment:
Nit: Open base level, not new level.

Reply:
changed teh comment to read opening base level.